app/lib/katello/event_daemon/runner.rb

```diff
@@ -54,8 +49,11 @@ def stop
 
         def start
           return unless runnable?
-          lockfile = File.open(lock_file, 'r')
-          begin
+
+          FileUtils.mkdir_p(tmp_dir)
+          FileUtils.touch(lock_file)
+
+          File.open(lock_file, 'r') do |lockfile|
             lockfile.flock(File::LOCK_EX)
             return if started? # ensure it wasn't started while we waited for the lock
```

Comment:
```
[2022-12-08T09:55:52.120Z] /home/jenkins/workspace/katello-pr-test/app/lib/katello/event_daemon/runner.rb:58:13: W: Lint/NonLocalExitFromIterator: Non-local exit from iterator, without return value. next, break, Array#find, Array#any?, etc. is preferred.
```

I don't see no iterator here?

Reply:
Suppose you could just return nil explicitly.

Reply:
*that* is what it complains about? ACAB!